src/main/java/gyro/google/compute/InstanceTemplateResource.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.InstanceProperties;
+import com.google.api.services.compute.model.InstanceTemplate;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Output;
+import gyro.core.resource.Resource;
+import gyro.core.scope.State;
+import gyro.core.validation.ConflictsWith;
+import gyro.core.validation.DependsOn;
+import gyro.core.validation.Regex;
+import gyro.core.validation.Required;
+import gyro.core.validation.ValidationError;
+import gyro.google.Copyable;
+
+/**
+ * Creates an Instance Template.
```

Comment:
To be consistent with other resources.

```suggestion
 * Creates an instance template.
```